### PR TITLE
Feature/alarms

### PR DESCRIPTION
**Pull request by marioncottard**
Adapted from https://github.com/baobabsoluciones/cornflow/pull/405.

### CODE REVIEW

File: cornflow-server/cornflow/config.py [hunk 1]
```diff
@@ -92,6 +95,7 @@ class Testing(DefaultConfig):
     AIRFLOW_PWD = os.getenv("AIRFLOW_PWD", "admin")
     OPEN_DEPLOYMENT = 1
     LOG_LEVEL = int(os.getenv("LOG_LEVEL", 10))
+    ALARMS_ENDPOINTS = 1
```
Comment (ggsdc) [inline cornflow-server/cornflow/config.py:98]: Should be 0 as default?

File: cornflow-server/cornflow/tests/unit/test_main_alarms.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+"""
+
+"""
+# TODO: Remove this
+import sys, os
+prev_dir = os.path.join(os.path.dirname(__file__), "..", "..", "..")
+print(prev_dir)
```
Comment (ggsdc) [inline cornflow-server/cornflow/tests/unit/test_main_alarms.py:7]: This print has to go out

File: cornflow-server/cornflow/tests/unit/test_main_alarms.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+sys.path.insert(1, prev_dir)
```
Comment (ggsdc) [inline cornflow-server/cornflow/tests/unit/test_main_alarms.py:8]: Why do you need this?

File: docker-compose.yml [hunk 4]
```diff
@@ -18,6 +18,7 @@ services:
       - CORNFLOW_DB_USER=cornflow
       - CORNFLOW_DB_PASSWORD=cornflow
       - CORNFLOW_DB=cornflow
+      - CF_ALARMS_ENDPOINT=1
```
Comment (ggsdc) [inline docker-compose.yml:21]: This should be zero

File: cornflow-server/cornflow/tests/unit/test_main_alarms.py (hunk 3 above)
Comment (marioncottard) [reply cornflow-server/cornflow/tests/unit/test_main_alarms.py]: Because this way I can run the tests directly with Pycharm

Comment (ggsdc) [reply cornflow-server/cornflow/tests/unit/test_main_alarms.py]: That is weird, you should be able to run the tests without the need for this.

**Review by ggsdc — APPROVED** [review]
Once is out of draft phase and in sync with development we can merge

**Pull request merged**